app.js

```diff
@@ -1,8 +1,253 @@
+
+  return createResource(
+    action.params,
+    settings,
+    GCCompute.BackendBucketsClient,
+    resource,
+  );
+}
+
+async function createSslCertificateFromJSON(action, settings) {
+  const resource = { sslCertificateResource: action.params.sslCertificateJSON };
+
+  return createResource(
+    action.params,
+    settings,
+    GCCompute.SslCertificatesClient,
+    resource,
+  );
+}
+
+async function createSslPolicyFromJSON(action, settings) {
+  const resource = { sslPolicyResource: action.params.sslPolicyJSON };
+
+  return createResource(
+    action.params,
+    settings,
+    GCCompute.SslPoliciesClient,
+    resource,
+  );
+}
+
+async function createTargetInstanceFromJSON(action, settings) {
+  const resource = {
+    targetInstanceResource: action.params.targetInstanceJSON,
+    zone: action.params.zone,
+  };
+
+  return createResource(
+    action.params,
+    settings,
+    GCCompute.SslPoliciesClient,
+    resource,
+  );
+}
+
+async function createTargetPoolFromJSON(action, settings) {
+  const resource = {
+    targetPoolResource: action.params.targetPoolJSON,
+    region: action.params.region,
+  };
+
+  return createResource(
+    action.params,
+    settings,
+    GCCompute.TargetPoolsClient,
+    resource,
+  );
+}
+
+async function deleteHealthCheck(action, settings) {
+  const resource = {
+    healthCheck: action.params.healthCheckName.value,
+  };
+  return deleteResource(
+    action.params,
+    settings,
+    GCCompute.HealthChecksClient,
+    resource,
+  );
+}
+
+async function deleteBackendService(action, settings) {
+  const resource = {
+    backendService: action.params.backendServiceName.value,
+  };
+  return deleteResource(
+    action.params,
+    settings,
+    GCCompute.BackendServicesClient,
+    resource,
+  );
+}
+
+async function deleteUrlMap(action, settings) {
+  const resource = {
+    urlMap: action.params.urlMapName.value,
+  };
+  return deleteResource(
+    action.params,
+    settings,
+    GCCompute.UrlMapsClient,
+    resource,
+  );
+}
+
+async function deleteTargetHttpProxy(action, settings) {
+  const resource = {
+    targetHttpProxy: action.params.targetHttpProxyName.value,
+  };
+  return deleteResource(
+    action.params,
+    settings,
+    GCCompute.TargetHttpProxiesClient,
+    resource,
+  );
+}
+
+async function deleteTargetHttpsProxy(action, settings) {
+  const resource = {
+    targetHttpsProxy: action.params.targetHttpsProxyName.value,
+  };
+  return deleteResource(
+    action.params,
+    settings,
+    GCCompute.TargetHttpsProxiesClient,
+    resource,
+  );
+}
+
+async function deleteForwardingRules(action, settings) {
+  const resource = {
+    forwardingRule: action.params.forwardingRuleName.value,
+  };
+  return deleteResource(
+    action.params,
+    settings,
+    GCCompute.GlobalForwardingRulesClient,
+    resource,
+  );
 }
 
 module.exports = {
-  hello,
+  createHealthCheckFromJSON,
```

Comment:
Almost lal of the `create` and all of the `delete` functions share the common code which differs with `GCCompute.*` and `resource` only.

Reply:
I revoke that. Too much OO perception